sapi_manifests/boray/template

```diff
@@ -72,6 +72,10 @@ tls_mode = "{{BORAY_DATABASE_TLS_MODE}}"
 tls_mode = "disable"
 {{/BORAY_DATABASE_TLS_MODE}}
 
+[zookeeper]
+path = "/manatee/{{SERVICE_NAME}}"
+zk_conn_str = "{{#ZK_SERVERS}}{{host}}:{{port}}{{^last}},{{/last}}{{/ZK_SERVERS}}"
```

Comment:
This is a nit, but we already have the context of zookeeper here from the `zookeeper` table name so maybe we could just call this `connection_string`? Once we pull it into an instance of `ConfigZookeeper` that will also provide the context for what the connection string pertains to.

Reply:
Done